server/src/api/routes/shipping/index.ts

```diff
@@ -0,0 +1,57 @@
+import { Router } from 'express';
+import middlewares from '@/api/middlewares';
+import {
+  handleGetShippings,
+  handlePostShipping,
+  handlePutShipping,
+  handleDeleteShipping,
+  handleSelectShipping,
+} from './shippingController';
+
+const shippingRouter = Router();
+
+export default (router: Router) => {
+  router.use('/shipping', shippingRouter);
+
+  shippingRouter.get(
+    '/',
+    middlewares.isAuth,
+    middlewares.isAccessToken,
+    middlewares.attachCurrentUser,
+    handleGetShippings,
+  );
+
```

Comment:
보다보니 너무 많은 인자를 get 함수에서 받는 느낌이 드네요...
비슷한 성격의 인자들은 객체로 묶어서 보내주는 것도 좋아보입니다.
다만 바꾸기 어려운 상황이라면 패쓰하셔도 됩니다 😅 

```suggestion
  shippingRouter.get(
    '/',
    {middlewares.isAuth, middlewares.isAccessToken, middlewares.attachCurrentUser},
    handleGetShippings,
  );

```

Reply:
으음,,,,, 그 부분은 미들웨어 설정하는 부분인데도 그렇게 사용 가능한가요?

Reply:
구조에 대한 파악 없이 함수만 봤을 때 느낀점입니다...! 
너무 많은 인자를 함수로 넘기는 것은 좋지 않다라고 들은 것 같아서요

<img width="1006" alt="스크린샷 2021-08-23 오후 3 56 54" src="https://user-images.githubusercontent.com/19240202/130403793-408b0251-5e30-4808-a909-ea54dcdaf9df.png">

Reply:
미들웨어 부분이라 변경이 가능한가 싶긴한데,  생각해 볼만한 부분인 것 같네요. 앞으로 코딩할때도 인자를 최대한 적게 넘기도록 노력해야 겠어요. 감사합니당:)

Reply:
넵, 맞습니다. 말씀해주신 내용은 옳습니다. 다만 get이라는 함수가 저희가 만든 함수가 아니라 express에서 제공하고 있는 메서드라,,,,

Reply:
만약에 줄인다고 한다면 여러 미들웨어를 하나의 미들웨어로 합칠 수 있는 방법에 대해서 알아봐야 할 것 같은데 이 부분은 추후에 생각해볼 문제인 것 같아요.

Reply:
아하... 이해했습니다 답변 감사합니다 :)